src/Tribe/Context.php

```diff
@@ -514,6 +514,11 @@ protected function query_var( array $query_vars, $default ) {
 		$value = $default;
 
 		global $wp_query;
+
+		if ( ! $wp_query instanceof \WP_Query ) {
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: ad2d4d6096da0fadada894270f189a10</sub>